eng/pipelines/common/xplat-setup.yml

```diff
@@ -124,6 +124,9 @@ jobs:
         ${{ if eq(parameters.osGroup, 'tvOS') }}:
           vmImage: 'macOS-10.14'
 
+        ${{ if eq(parameters.osGroup, 'Browser') }}:
+          vmImage: 'ubuntu-latest'
```

Comment:
why is this needed?

Reply:
This is needed because when you create new pipelines you can't choose a default pipeline anymore other than "Hosted Azure" which forces you to specify a vmImage. In this case Browser was using Ubuntu16.04 hosted for all other pipelines now it will use hosted `ubuntu-latest`